ts/sagas/backendInfo.ts

```diff
@@ -28,7 +29,16 @@ function* backendInfoWatcher(): IterableIterator<Effect> {
     return new Promise((resolve, _) =>
       backendPublicClient
         .getServerInfo({})
-        .then(resolve, () => resolve(undefined))
+        .then(resolve, () =>
+          resolve(
+            left([
```

Comment:
you could try to use a more coincise way to create [Errors](https://github.com/gcanti/io-ts/blob/4d13500147bfd654c3db2d6f18a02334fbae44db/src/index.ts#L37)
```javascript
const error: Errors = [
          {
            context: [],
            message: "error message",
            value: your_value
          }
        ];
        return resolve(left(error));
```

Reply:
Fix in: ba56f18